vividus-extension-selenium/src/main/java/org/vividus/selenium/WebDriverManagerContext.java

```diff
@@ -54,4 +63,10 @@ public void reset(WebDriverManagerParameter parameter)
     {
         testContext.remove(parameter.getContextKey());
     }
+
+    @Subscribe
```

Comment:
is `WebDriverManagerContext` bean marked as `lazy-init=false`?

Reply:
I'll add `<bean class="org.vividus.selenium.WebDriverManagerContext" lazy-init="false" />`  in `spring.xml`